EFCore/MAUI/Readme.md

```diff
@@ -197,7 +197,31 @@ This example demonstrates how to use our [Secured WebAPI](https://docs.devexpres
    ```cs
    public virtual MediaDataObject Photo { get; set; }
    ```
-## Step 5.  [Create A Predefined Static Report](https://docs.devexpress.com/eXpressAppFramework/113645/shape-export-print-data/reports/create-predefined-static-reports) to display the Post business objects.
```

Comment:
>This example demonstrates how to use our [Secured WebAPI](https://docs.devexpress.com/eXpressAppFramework/113366/concepts/security-system/security-system-overview) from a MAUI application using EF Core for data access. We will create new endpoints around a `Post` Business object and will use from MAUI application to consume them. We will demo how to `Authenticate`,`List Posts`, `Create new Posts` `Display Post Reports`, `Archive a Post`, `Display a Post Author Photo`.

Please replace to:

>This example demonstrates how to use our [Web API Service](https://docs.devexpress.com/eXpressAppFramework/403394/backend-web-api-service) with a mobile .NET MAUI application using EF Core for data access. Our .NET MAUI application will use API endpoints for the `Post` business object to implement the following common scenarios:
 - Authenticate and authorize users from different roles;
 - List existing Post records;
 - Create new Post records;
 - Display a report based on Post records;
 - Archive a Post record;
 - Display a photo of a Post author.

Reply:
>[VisibleInReports][DomainComponent]
     public class Post:IXafEntityObject,IObjectSpaceLink {
     
Please remove the DomainComponent from the code and readme - it is NOT needed and only complicates pur solution and user understanding.

Folks, if you have time, please rework the Post class completely so that it becomes more elegant and consise. Take special note that I replaced the integer key PostId with the Guid key ID, so you will need to modify at least the AuthorPhoto at Archive methods. If it is too much to redo and retest, then leave it as is. Thanks.

```
[VisibleInReports]
public class Post : BaseObject {
    public virtual string Title { get; set; }
    public virtual string Content { get; set; }
    public virtual ApplicationUser Author { get; set; }
    public override void OnCreated() {
        base.OnCreated();
        Author = ObjectSpace.FirstOrDefault<ApplicationUser>(u => u.ID == (Guid)CurrentUserIdOperator.CurrentUserId());
    }
}
```

Reply:
>## Step 1. WebAPI initialization: [Create WebAPI project using the XAF solution wizard](https://docs.devexpress.com/eXpressAppFramework/403401/backend-web-api-service/create-new-application-with-web-api-service).

A URL in the header looks ugly - please put the link into the boxy.

Reply:
>## Step 4. Work on the Business objects.

Please place this section earlier, right after you created the Web API Service - it is strange that you mention the Post class in other sections before declaring it.
Also, rename this header to "Step 1. Declare a data model."


>## Step 2. Create the WebAPI endpoints.

Rename to "## Step 2. Define built-in and custom the Web API endpoints."
I would also move "6. Modify the Startup.cs to register the `build-in CRUD endpoints` for the Post object." to the beginning of this section.

>1. Add a `CustomEndPointController` inside the `WebAPI/API` directory, injecting the `ISecurityProvider` and `IObjectSpaceFactory` to request user permissions and retrieve database data from the endpoints we will create later.

Add a link to the explanation, please: https://docs.devexpress.com/eXpressAppFramework/403669/data-manipulation-and-business-logic/access-object-space-security-and-caption-helper-in-asp-net-core-environment